packages/main/src/SegmentedButtonItem.js

```diff
@@ -11,68 +11,55 @@ const metadata = {
 	tag: "ui5-segmented-button-item",
 	properties: /** @lends  sap.ui.webcomponents.main.SegmentedButtonItem.prototype */ {
 		/**
-		 * Defines the <code>ui5-button</code> design.
+		 * <b>Note:</b> The property is inherited and not supported. If set, it won't take any effect.
 		 *
-		 * <br><br>
-		 * <b>Note:</b>
-		 *
-		 * <ul>
-		 * <li><code>Default</code></li>
-		 * <li><code>Emphasized</code></li>
-		 * <li><code>Positive</code></li>
-		 * <li><code>Negative</code></li>
-		 * <li><code>Transparent</code></li>
-		 * </ul>
-		 *
-		 * @type {ButtonDesign}
-		 * @defaultvalue "Default"
-		 * @private
+		 * @public
 		 */
 		design: {
 			type: ButtonDesign,
 			defaultValue: ButtonDesign.Default,
 		},
 
 		/**
-		 * Defines whether the icon should be displayed after the <code>ui5-button</code> text.
+		 * <b>Note:</b> The property is inherited and not supported. If set, it won't take any effect.
 		 *
-		 * @type {boolean}
-		 * @defaultvalue false
-		 * @private
+		 * @public
 		 */
 		iconEnd: {
 			type: Boolean,
 		},
 
 		/**
-		 * Defines the size of the icon inside the <code>ui5-button</code>.
+		 * <b>Note:</b> The property is inherited and not supported. If set, it won't take any effect.
 		 *
-		 * @type {string}
-		 * @defaultvalue undefined
-		 * @private
+		 * @public
 		 */
 		iconSize: {
 			type: String,
 			defaultValue: undefined,
 		},
 
 		/**
-		 * When set to <code>true</code>, the <code>ui5-button</code> will
-		 * automatically submit the nearest form element upon <code>press</code>.
-		 * <br><br>
-		 * <b>Important:</b> For the <code>submits</code> property to have effect, you must add the following import to your project:
-		 * <code>import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";</code>
+		 * <b>Note:</b> The property is inherited and not supported. If set, it won't take any effect.
 		 *
-		 * @type {boolean}
-		 * @defaultvalue false
-		 * @private
+		 * @public
 		 */
 		submits: {
 			type: Boolean,
 		},
 
+		/**
+		 * <b>Note:</b> The property is inherited and not supported. If set, it won't take any effect.
+		 *
+		 * @public
+		 */
+		pressed: {
```

Comment:
If `pressed` is not supported how the the developers would be able to preselect a segmented button item

```html
<ui5-segmented-button>
<ui5-segmented-button-item>first
<ui5-segmented-button-item pressed>second
<ui5-segmented-button-item>third
```